stable/yugaware/templates/statefulset.yaml

```
@@ -134,9 +137,13 @@ spec:
           image: {{ include "full_image" (dict "containerName" "prometheus" "root" .) }}
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           {{- if (not .Values.ocpCompatibility.enabled) }}
+          {{- if .Values.securityContext.enabled }}
+          securityContext: {{- .Values.securityContext.containerLevel | toYaml | nindent 12 }}
+          {{- else }}
           securityContext:
             runAsUser: 0
           {{- end }}
+          {{- end }}
```

```suggestion
          {{- if .Values.securityContext.enabled }}
          securityContext: {{- .Values.securityContext.containerLevel | toYaml | nindent 12 }}
          {{- else }}
          {{- if (not .Values.ocpCompatibility.enabled) }}
          securityContext:
            runAsUser: 0
          {{- end }}
          {{- end }}
```

With this, we still allow one to set own securityContext when running on OCP as well.